app/javascript/lib/validation/shared-validation.js

```diff
@@ -2,6 +2,22 @@
 * @fileoverview Functions used in multiple file types validation
 */
 
+// from lib/assets/metadata_schemas/alexandria_convention_schema.json
+// (which in turn is from scp-ingest-pipeline/schemas)
+export const REQUIRED_CONVENTION_COLUMNS = [
```

Comment:
Thinking out loud... I know this is the same as we always did previously, but I wonder if it's possible to source this directly from `lib/assets/metadata_schemas/alexandria_convention/alexandria_convention_schema.json`?

Reply:
Good idea!  This required some configuration as that file is outside our frontend root (app/javascript), but it works. 

Consolidating here helps avoid problems if we update required headers.